src/Microsoft.Health.Dicom.Core/Features/Retrieve/RetrieveResourceService.cs

```diff
@@ -219,6 +219,9 @@ public async Task<RetrieveResourceResponse> GetInstanceResourceAsync(RetrieveRes
         }
         _logger.LogInformation("Downloading the entire instance for frame parsing");
 
+        // Get file properties again for transcoding
+        instance = await GetInstanceMetadata(instanceIdentifier, isOriginalVersion: false, cancellationToken);
```

Comment:
No,  we have cache for getInstance method. I made that call default to request original since that is the maximum number of calls we get. So on transcoding, I go and fetch the latest file properties.